content/en/docs/concepts/observability-primer.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Observability Primer
+weight: 9
+---
+
+Before digging into OpenTelemetry, it is important to understand some core
+concepts first.
+
+## What is Observability?
+
+Observability lets us understand a system from the outside, by letting us ask
+questions about that system without knowing its inner workings. Furthermore,
+allows us to easily troubleshoot and handle novel problems (i.e. “unknown
```

Comment:
minor, the double quotes here are mixed of UNICODE and ASCII (the double quotes on "unknown unknowns" are non-ASCII, while the ones on "Why is this happening" are ASCII).

Reply:
it seems there are more places, might be a good idea to put some CI guard.

Reply:
Oh yeah...I just did a quick search and it's happening all over the place...

Fixed it in the pages I've touched.